candle-book/src/guide/installation.md

```diff
@@ -1 +1,24 @@
 # Installation
+
+Start by creating a new app:
+
+```bash
+cargo new myapp
+cd myapp
+cargo add --git https://github.com/LaurentMazare/candle.git candle
+```
+
+At this point, candle will be built **without** CUDA support.
+To get CUDA support use the feature `cuda`
```

Comment:
the feature `cuda` -> the `cuda` feature?

Reply:
Done.